Hermes/handlers.py

```diff
@@ -143,3 +146,43 @@ def post(self):
       TASK_STATUS_LOCK.release()
 
     self.set_status(hermes_constants.HTTP_Codes.HTTP_OK)
+
+class NodeStatsHandler(RequestHandler):
+  """ Main handler class. """
+
+  # The path for this handler.
+  PATH = "/node_stats"
+
+  def initialize(self, STATS, secret):
+    self.STATS = STATS
+    self.secret = secret
+    self.BAD_SECRET_MESSAGE = {'success': False, 'reason': 'bad secret'}
```

Comment:
@cdonati the message part I'll change, but the STATS attribute is so that it can use the same STATS variable that's being used in hermes, that was the way I could use the variable across files supposedly.